.github/workflows/build.yml

```diff
@@ -140,9 +140,16 @@ jobs:
               `;
             }, '' );
             const commentBody = `${ TEST_COMMENT_INDICATOR }
-            Are you an Automattician? You can now test your Pull Request on WordPress.com. On your sandbox, run
+            Are you an Automattician? Please test your changes on WordPress.com.
+
+            - To test on WoA, install [the Beta plugin](https://jetpack.com/download-jetpack-beta/), go to Jetpack > Jetpack Beta > Jetpack and enable the \`${ BRANCH_NAME }\` branch.
+            - To test on Simple, run the following command on your sandbox:
             ${ commands }
-            to get started. More details: p9dueE-5Nn-p2`;
+
```

Comment:
As a WordPress.com developer, it would be helpful to have this information easily accessible, instead of hidden behind a link.

Reply:
> it might be better to just include a "p9dueE-5Nn-p2"-style doc ref instead of making this comment extremely detailed. 

That's also what I had in mind. I worry that if the comment gets too long, folks will ignore it.

I was updating some of our FieldGuide pages with that in mind, and would suggest something like this:

```diff
diff --git a/.github/workflows/build.yml b/.github/workflows/build.yml
index 8afb3754a6..cdf40a472d 100644
--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -146,10 +146,11 @@ jobs:
             - To test on Simple, run the following command on your sandbox:
             ${ commands }
 
-            *Interested in more testing tips?*
+            *Interested in more tips and information?*
 
             - In your local development environment, use the \`jetpack rsync\` command to sync your changes to a WoA dev blog.
-            - Find out more about testing on WordPress.com Simple here: p9dueE-5Nn-p2`;
+            - Read more about our development workflow here: PCYsg-eg0-p2
+            - Figure out **when your changes will be shipped to customers** here: PCYsg-eg5-p2`;
             await github.rest.issues.updateComment( {
               owner: context.repo.owner,
               repo: context.repo.repo,

```

What do you think?

Reply:
> Figure out **when your changes will be shipped to customers** here: PCYsg-eg5-p2

This doesn't actually say _when_, though:

<img width="775" alt="image" src="https://github.com/Automattic/jetpack/assets/36432/672a0aca-4f4e-42ce-b9a8-eb7596cad70f">

Nor does PCYsg-Jjm-p2. And `#jetpack-releases` isn't linked in the first document.

Atomic makes it pretty clear, and this clarity is helpful:

<img width="595" alt="image" src="https://github.com/Automattic/jetpack/assets/36432/b4691c8a-0870-4d7f-bacf-d7cc1ab83423">

Reply:
It says as close to when as we have information available. It's not like we have a specific time to do any of it, particularly since it's not always a person in the same timezone being on the rotation to do it.

Plus some of what's there has changed pretty recently, and might do so again in the near future. As I mentioned, it's easier to update that doc page (which will have to be updated anyway) than to have to make more commits in here.

Reply:
We can agree to disagree, then. I think hiding information away in documents makes it less accessible, particularly when the documents themselves get out of date.

Putting the information in the GitHub bot message:
1. Makes the information more accessible and obvious.
2. Helps ensure it will be corrected sooner when it's out of date (because folks are looking at it every day).

Reply:
I committed the above in 76cd497d2317007b86fec2cb280b8ab6affb3348. We can see how this change is received, and iterate in follow-up PRs based on the feedback we'll get.